OpenRA.Mods.Common/Traits/AmmoPool.cs

```diff
@@ -66,35 +66,40 @@ public class AmmoPool : INotifyCreated, INotifyAttack, IPips, ISync
 		[Sync]
 		public int RemainingTicks;
 
-		[Sync]
 		int currentAmmo;
 
+		[Sync]
+		public int CurrentAmmo
```

Comment:
Declare this as `{ get; private set; }` and remove `currentAmmo` completely.

Reply:
Done.